src/backend/src/services/projects.services.ts

```diff
@@ -107,12 +123,38 @@ export default class ProjectsService {
         projectNumber: maxProjectNumber + 1,
         workPackageNumber: 0,
         name,
+        projectLeadId,
+        projectManagerId,
+        links: links
+          ? {
+              createMany: {
+                data: links.map((link) => ({ ...link, creatorId: user.userId }))
+              }
+            }
+          : undefined,
         project: {
           create: {
             summary,
             teams: {
               connect: teamIds.map((teamId) => ({ teamId }))
-            }
+            },
+            budget: budget ?? undefined,
+            rules: rules ?? undefined,
+            goals: goals
+              ? {
+                  createMany: { data: goals }
+                }
+              : undefined,
+            features: features
+              ? {
+                  createMany: { data: features }
+                }
+              : undefined,
+            otherConstraints: otherConstraints
+              ? {
+                  createMany: { data: otherConstraints }
+                }
+              : undefined
```

Comment:
This is what I had to do because the fields are optional

Reply:
view slack